src/controllers/paymentController.js

```diff
@@ -33,3 +33,31 @@ export const registerPayment = async (req, res) => {
   }
 }
 
+export const deletePayment = async (req, res) => {
+
+  try {
+    const { id } = req.params;
+    const payment = await Payment.findById(id);
+    if (!payment) {
+      logger.error('Payment not found', {
+        method: req.method,
```

Comment:
I recommend you to follow the new log structure, [here](https://github.com/FIS2425/appointment-svc/commit/434d0c1eb22a72cfa4df0f2e4b324d117f2cbd01) is an example.